lib/sen_tweet_web/live/stats_live.html.leex

```diff
@@ -52,31 +55,31 @@
 
 <div class="columns">
   <div class="column is-two-thirds">
-    <%= @hourly.svg %>
+    <%= hourly.svg %>
   </div>
 
   <div class="column">
     <div class="box">
       <h1 class="title has-text-centered">Tweet Type</h1>
       <div class="buttons are-large">
-        <button class="button is-light is-fullwidth is-outlined <%= is_selected("hourly_text", @hourly.filter.type, @hourly.filter.weight) %>" phx-click="hourly_text">Text</button>
+        <button class="button is-light is-fullwidth is-outlined <%= is_selected("hourly_text", hourly.filter["type"], hourly.filter["weight"]) %>" phx-click="text" phx-value-stream="<%= stream %>" phx-value-interval="hourly" phx-value-filter="type">Text</button>
```

Comment:
We can add `phx-value-*` to send data to the `phx-click` events.

For example `phx-value-interval="hourly"` adds `%{"interval" => "hourly"}` to the `params` in the `handle_event` callback.

I was able to remove a lot of duplication by sending specific values.